src/main/config/config.h

```diff
@@ -29,7 +29,6 @@
 
 typedef enum {
     CONFIGURATION_STATE_DEFAULTS_BARE = 0,
-    CONFIGURATION_STATE_DEFAULTS_CUSTOM,
```

Comment:
You also need to get rid of one element here, if I'm not mistaken
https://github.com/betaflight/betaflight/blob/master/src/main/cli/cli.c#L283

Reply:
maybe also change `CONFIGURATION_STATE_DEFAULTS_BARE` to `CONFIGURATION_STATE_UNCONFIGURED` to be consistent?

Reply:
Thanks. Fixed.